primefaces/src/main/java/org/primefaces/component/overlaypanel/OverlayPanel.java

```diff
@@ -53,6 +53,10 @@ public class OverlayPanel extends OverlayPanelBase {
             .build();
     private static final Collection<String> EVENT_NAMES = BEHAVIOR_EVENT_MAPPING.keySet();
 
+    public static OverlayPanelRenderer getDefaultRenderer(FacesContext context) {
+        return (OverlayPanelRenderer) context.getRenderKit().getRenderer(COMPONENT_FAMILY, DEFAULT_RENDERER);
```

Comment:
I would not make it static
make it non static and call it getRenderer() which does:
getFacesContext().getRenderKit().getRenderer(getFamily(), getRendererType());